### PR TITLE
Feat/onclick header tab

### PR DESCRIPTION
**Pull request by varun257**
### Description of the Change

add on-click event for header-tabs-item
add test case and example

### Benefits

More options

### Possible Drawbacks

None

### Applicable Issues

Helps resolve MBE-284

### CODE REVIEW

File: packages/oui-slideshow/README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@
 | `loading`     | boolean   | <?        | no                | `true`, `false`   | `false`   | display loader flag
 | `loop`        | boolean   | <?        | no                | `true`, `false`   | `false`   | whether the component should cycle continuously
 | `on-dismiss`  | function  | &         | no                | n/a               | n/a       | dismiss callback
+| `on-next`     | function  | &         | no                | n/a               | n/a       | handler triggered when on click of next slide
```
Comment (JeremyDec) [inline packages/oui-slideshow/README.md]: You should remove this (remains of your `oui-slideshow` changes)

File: packages/oui-header-tabs/README.md [hunk 2]
```diff
@@ -97,6 +97,7 @@
 | `active`      | boolean   | <?        | no                | `true`, `false`   | `false`   | manual active flag
 | `disabled`    | boolean   | <?        | yes               | `true`, `false`   | `false`   | disabled flag
 | `external`    | boolean   | <?        | yes               | `true`, `false`   | `false`   | external link flag
+| `on-click`    | function  | &         | no                | n/a               | n/a       | handler triggered when clicked
```
Comment (JeremyDec) [inline packages/oui-header-tabs/README.md]: You can simplify the description this way : `click handler` or `click event handler`

**Comment by JeremyDec** [comment]
@varun257 Also, don't hesitate to drop your first commit, in order to have a clean commit history

File: packages/oui-header-tabs/README.md (hunk 2 above)
Comment (FredericEspiau) [reply packages/oui-header-tabs/README.md]: I like `click handler` better :)

**Pull request merged**